src/main/resources/db/migration/V2__indexes.sql

```diff
@@ -0,0 +1,29 @@
+create index as_acquirer_id_index
+    on acquirer_subscription (acquirer_id);
+
+create index as_oauth_client_id_index
+    on acquirer_subscription (oauth_client_id);
+
+create index as_event_type_index
+    on acquirer_subscription (event_type);
+
+create index asef_acquirer_subscription_id_index
+    on acquirer_subscription_enrichment_field (acquirer_subscription_id);
+
+create index ed_acquirer_subscription_id_index
+    on event_data (acquirer_subscription_id);
+
+create index ed_deleted_at_index
+    on event_data (deleted_at);
+
+create index ed_polling_index
+    on event_data (when_created, acquirer_subscription_id, deleted_at);
```

Comment:
I'm never quite sure when partial indexes are worth it, but it seems like here might be a good place? For `deleted_at IS NULL`?